src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -53,9 +57,25 @@ const DetailsSidebar = ({
     }
 
     const onDescriptionChangeEditable = getProp(file, 'permissions.can_rename') ? onDescriptionChange : undefined;
+    let sharedLinkExpiration = new Date(getProp(file, 'shared_link.unshared_at'));
+
+    if (sharedLinkExpiration) {
```

Comment:
Move this logic inside `SidebarNotices`
Also move the `<sidebarsection>` inside there.

So here, the UI looks like
`{hasNotices && <SidebarNotices file={file} />}

`hasNotices` should also come from the parent component, like other `has*` properties.
Just replicate those, default being false.

And then to show shared link expiration or not, that logic should be inside `SidebarNotices`

Reply:
Is the reason for making it a passed in prop so that people can toggle it on and off? By default, if we have notices wouldn't we want to show them?

Reply:
Yeah, the sidebar will be all with togglable sections